client/my-sites/plugins/plugin-activate-toggle/test/index.jsx

```diff
@@ -43,6 +43,7 @@ describe( 'PluginActivateToggle', function() {
 	} );
 
 	beforeEach( function() {
+		this.timeout( 10 * 1000 );
```

Comment:
10 seconds seems to be a lot! It is 3 times longer than the new single test runner we work on atm :D